app/scripts/models/auth_brokers/fx-sync-channel.js

```diff
@@ -182,6 +182,15 @@ define(function (require, exports, module) {
     },
 
     afterResetPasswordConfirmationPoll (account) {
+      // We wouldn't expect `customizeSync` to be set when completing
+      // a password reset, but the field must be present for the message
```

Comment:
`for the message to be sent for the login message to be sent`. Seems like only the 2nd half of the sentence is useful.